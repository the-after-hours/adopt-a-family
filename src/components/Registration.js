import React from 'react';
import Form from './Form';

const Registration = (props) => {
  return (
    <div>
      <Form />
    </div>
  );
};

export default Registration;