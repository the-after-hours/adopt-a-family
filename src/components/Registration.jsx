import React from 'react';
import Form from './Form';

const Registration = (props) => (
  <div>
    <Form />
  </div>
);

export default Registration;
