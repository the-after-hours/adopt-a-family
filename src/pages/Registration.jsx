import React from 'react';
import Form from '../components/Form';

const Registration = (props) => (
  <div>
    <Form />
  </div>
);

export default Registration;
