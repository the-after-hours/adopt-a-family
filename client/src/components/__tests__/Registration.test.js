import React from 'react';
import Registration from '../Registration';
import renderer from 'react-test-renderer';

describe('Registration', () => {
  test('Registration snapshot', () => {
    const component = renderer.create(<Registration />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
