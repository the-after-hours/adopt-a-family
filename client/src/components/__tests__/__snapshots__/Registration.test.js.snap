// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Registration Registration snapshot 1`] = `
<div>
  <div>
    <div
      className="contact"
    >
      <form>
        <span
          className="contact-label"
        >
          First Name:
        </span>
        <input
          className={undefined}
          maxLength={undefined}
          name="firstName"
          onChange={undefined}
          pattern={undefined}
          placeholder="First Name"
          size={undefined}
          title={undefined}
          type="text"
        />
        <br />
        <span
          className="contact-label"
        >
          Middle Initial:
        </span>
        <input
          className={undefined}
          maxLength="1"
          name="middleInitial"
          onChange={undefined}
          pattern={undefined}
          placeholder="Middle Initial"
          size={undefined}
          title={undefined}
          type="text"
        />
        <br />
        <span
          className="contact-label"
        >
          Last Name:
        </span>
        <input
          className={undefined}
          maxLength={undefined}
          name="lastName"
          onChange={undefined}
          pattern={undefined}
          placeholder="Last Name"
          size={undefined}
          title={undefined}
          type="text"
        />
        <br />
        <span
          className="contact-label"
        >
          Address
        </span>
        <input
          className={undefined}
          maxLength={undefined}
          name="address"
          onChange={undefined}
          pattern={undefined}
          placeholder="Address"
          size={undefined}
          title={undefined}
          type="text"
        />
        <br />
        <span
          className="contact-label"
        >
          Account Type:
        </span>
        <select
          name="accountType"
        >
          <option
            value="donator"
          >
            Donator
          </option>
          <option
            value="family"
          >
            Family
          </option>
          <option
            value="organizer"
          >
            Organizer
          </option>
        </select>
        <br />
        <span
          className="contact-label"
        >
          Email:
        </span>
        <input
          className={undefined}
          maxLength={undefined}
          name="email"
          onChange={[Function]}
          pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\\\\.[a-z]{2,3}$"
          placeholder="Email"
          size={undefined}
          title={undefined}
          type="email"
        />
        <br />
        <span
          className="contact-label"
        >
          Confirm Email
        </span>
        <input
          className="emailInvalid"
          maxLength={undefined}
          name="confirmEmail"
          onChange={[Function]}
          pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\\\\.[a-z]{2,3}$"
          placeholder="Confirm Email"
          size={undefined}
          title={undefined}
          type="email"
        />
        <br />
        <span
          className="contact-label"
        >
          Password:
        </span>
        <input
          className={undefined}
          maxLength={undefined}
          name="password"
          onChange={undefined}
          pattern=".{6,}"
          placeholder="Password"
          size={undefined}
          title="Must be at least 6 characters."
          type="password"
        />
        <br />
        <span
          className="contact-label"
        >
          Confirm Password
        </span>
        <input
          className={undefined}
          maxLength={undefined}
          name={undefined}
          onChange={undefined}
          pattern=".{6,}"
          placeholder="Confirm Password"
          size={undefined}
          title="Must be at least 6 characters"
          type="password"
        />
        <br />
        <button
          onClick={[Function]}
          type="submit"
        >
          SUBMIT
        </button>
      </form>
    </div>
  </div>
</div>
`;
